packages/@lwc/engine/src/framework/decorators/__tests__/wire.spec.ts

```diff
@@ -9,82 +9,14 @@ import { createElement, LightningElement, registerDecorators } from '../../main'
 import wire from '../wire';
 
 const emptyTemplate = compileTemplate(`<template></template>`);
+const WireAdapter = class EchoWireAdapter {
+    update() {}
+    connect() {}
+    disconnect() {}
+};
 
 describe('wire.ts', () => {
     describe('integration', () => {
-        it('should support setting a wired property in constructor', () => {
-            expect.assertions(3);
-
-            const o = { x: 1 };
-            class MyComponent extends LightningElement {
-                constructor() {
-                    super();
-                    expect('foo' in this).toBe(true);
-                    this.foo = o;
-
-                    expect(this.foo).toEqual(o);
-                    expect(this.foo).not.toBe(o);
-                }
-            }
-            registerDecorators(MyComponent, {
-                wire: { foo: {} },
-            });
-
-            const elm = createElement('x-foo', { is: MyComponent });
-            document.body.appendChild(elm);
-        });
-
-        it('should support wired properties', () => {
-            expect.assertions(2);
-
-            const o = { x: 1 };
-            class MyComponent extends LightningElement {
-                injectFoo(v) {
-                    this.foo = v;
-                    expect(this.foo).toEqual(o);
-                    expect(this.foo).not.toBe(o);
-                }
-            }
-            registerDecorators(MyComponent, {
-                wire: { foo: {} },
-                publicMethods: ['injectFoo'],
-            });
-
-            const elm = createElement('x-foo', { is: MyComponent });
-            document.body.appendChild(elm);
-            elm.injectFoo(o);
-        });
-
-        it('should make wired properties reactive', () => {
-            let counter = 0;
-            class MyComponent extends LightningElement {
-                injectFoo(v) {
-                    this.foo = v;
-                }
-                constructor() {
-                    super();
-                    this.foo = { x: 1 };
-                }
-                render() {
-                    counter++;
-                    this.foo.x;
-                    return emptyTemplate;
-                }
-            }
-            registerDecorators(MyComponent, {
-                wire: { foo: {} },
-                publicMethods: ['injectFoo'],
-            });
-
-            const elm = createElement('x-foo', { is: MyComponent });
-            document.body.appendChild(elm);
-            elm.injectFoo({ x: 2 });
-
-            return Promise.resolve().then(() => {
-                expect(counter).toBe(2);
-            });
-        });
-
         it('should make wired properties as readonly', () => {
```

Comment:
@caridy i didn't fully understood this test, therefore i leaved for you to review.